038_router_L125.js

```
@@ -0,0 +1,344 @@
+        logger.info(e); 
+    });
+});
+
+
+route.get('/downloadSong',function(req,res,next){
+    co(function*(){
+        var policy = new qiniu.rs.GetPolicy();
+        var downloadUrl = policy.makeRequest(req.query.url);
+        return Config.SUCCESS(downloadUrl);
+    }).then(function(data){
+        res.json(data);
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)